modules/reindex/src/test/java/org/elasticsearch/reindex/AsyncBulkByScrollActionTests.java

```diff
@@ -506,9 +511,6 @@ public boolean isCancelled() {
         };
         action.setScroll(scrollId());
 
-        // Set the base for the scroll to wait - this is added to the figure we calculate below
-        firstSearchRequest.scroll(timeValueSeconds(10));
-
```

Comment:
Moved up to apply the change before search request is cloned inside of DummyAsyncBulkByScrollAction constructor